package.json

```diff
@@ -39,15 +39,15 @@
     "coverage": "codecov"
   },
   "peerDependencies": {
-    "react": "^0.14.0 || ^15.0.0-0 || ^16.0.0-0",
+    "react": "^16.4.0-0",
     "redux": "^2.0.0 || ^3.0.0 || ^4.0.0-0"
   },
   "dependencies": {
     "hoist-non-react-statics": "^2.5.5",
     "invariant": "^2.2.4",
     "loose-envify": "^1.1.0",
-    "prop-types": "^15.6.1",
```

Comment:
This needs to remain as a plain dependency for the UMD dev build. 

Let's rip this kind of stuff out of this PR for now. I'd rather keep it to a minimal amount of changes in case there are other PRs merged in in the meantime. We can always do it later. I want to switch us to preset-env and upgrade to Babel 7 anyways.

Reply:
I'll do this change next.

Reply:
Hi, I just checked, and the UMD plain build has PropTypes in it, because we have it as a devDependency, and it is used by rollup to make the development UMD bundle.

In any case, it was trivial to add this, so it will be trivial to add it back in.